pkg/cmd/openshift-sdn-controller/interfaces.go

```diff
@@ -1,14 +1,17 @@
 package openshift_sdn_controller
 
 import (
+	"strings"
 	"time"
 
+	configv1 "github.com/openshift/api/config/v1"
+	osdcnclient "github.com/openshift/client-go/cloudnetwork/clientset/versioned"
+	osdcninformer "github.com/openshift/client-go/cloudnetwork/informers/externalversions"
```

Comment:
"osdcn" is a pretty non-obvious abbreviation for "cloud network". (Like, where does the "d" come from??) How about just `cloudnetworkclient`, `cloudnetworkinformer`? Yeah, it's kind of long, but it's not like you have to write it out _that_ many times...

("osdn" as an abbreviation for "openshift-sdn" is kind of weird itself, but we've kept it around to make it easier to distinguish OpenShift network API stuff (`"github.com/openshift/api/network/v1"`) from Kubernetes network API stuff (`"k8s.io/api/networking/v1"`), because if it was just `networkv1` and `networkingv1`, that would be kind of terrible.)

Reply:
Ah, I thought `osd` was a legacy acronym for "OpenShift something", and wanted to keep that in line hence "OpenShift something cloud-network". I will rename it.